server/store/session-store.js

```diff
@@ -66,6 +66,8 @@ class SessionStoreInternal extends SessionStore {
             if (session.closeId === closeId) {
                 return session 
             }
+            else if (session.closeId != closeId)
```

Comment:
just use `else`. If `session.closeId` is not equals to `closeId` then `else` block will be called anyway. We don't need additional condition.

Reply:
done